client/question/components/LearnerViewDirective.js

```diff
@@ -118,11 +118,17 @@ tie.directive('learnerView', [function() {
                 <div class="tie-code-auto-save" ng-class="{'night-mode': isInDarkMode}" ng-show="autosaveTextIsDisplayed">
                   Saving code...
                 </div>
+                <button class="tie-run-button tie-button tie-button-green"
+                    ng-class="{'active': !nextButtonIsShown}"
+                    ng-click="submitCode(editorContents.code)"
+                    ng-disabled="nextButtonIsShown">
+                  I think I'm done
+                </button>
                 <button class="tie-run-button tie-button tie-button-blue protractor-test-run-code-btn"
```

Comment:
protractor-test-run-code-btn doesn't apply to the "I think I'm done" button?

Reply:
Yes, this is deliberate. For determinism, I want the e2e test to trigger a specific button, not a random choice between the two.

Reply:
OK